src/debugger/debugger.c

```diff
@@ -988,12 +999,16 @@ PHP_FUNCTION(xdebug_break)
 {
 	RETURN_FALSE_IF_MODE_IS_NOT(XDEBUG_MODE_STEP_DEBUG);
 
+	xdebug_rebuild_stack_if_disabled();
```

Comment:
If we stop on a call to the xdebug_break() function we rebuild the stack in case we connect to a client and it requests some data about the current execution point